example/cmake/FindLibOSRM.cmake

```diff
@@ -0,0 +1,39 @@
+# - Try to find LibOSRM
+# Once done this will define
+#  LibOSRM_FOUND - System has LibOSRM
+#  LibOSRM_INCLUDE_DIRS - The LibOSRM include directories
+#  LibOSRM_LIBRARIES - The libraries needed to use LibOSRM
+#  LibOSRM_DEFINITIONS - Compiler switches required for using LibOSRM
```

Comment:
What about user-provided hints? Like, what if I install libosrm locally and want to pass a hint to this cmake module?